Scripts/script-http.yml

```diff
@@ -0,0 +1,90 @@
+commonfields:
+  id: http
+  version: -1
+name: http
```

Comment:
script description is missing, should have good explanation about the script and some examples.